Python/sysmodule.c

```diff
@@ -96,6 +96,80 @@ PySys_SetObject(const char *name, PyObject *v)
         return PyDict_SetItemString(sd, name, v);
 }
 
+static PyObject *
+sys_breakpointhook(PyObject *self, PyObject *args, PyObject *keywords)
```

Comment:
Do you want to allow users to modify PYTHONBREAKPOINT env var at runtime? Why not checking PYTHONBREAKPOINT once at startup, somewhere in Py_Initialize()?

Reply:
Yes, we do want to allow that.  Guido even pointed out that use case in one of his messages, and I agree.